WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -386,6 +386,21 @@ <h4>Where does my reduced data go?</h4>
                     </p>
                 </div>
             </section>
+            <section class="help-topic panel panel-default" data-category="question">
+                <div class="panel-heading" id="reduction-statuses">
+                    <h3>What do the reduction statuses mean?</h3>
+                </div>
+                <div class="panel-body">
+                    There are 5 different states a reduction can be in:
+                    <ul>
+                        <li>Queued - This is when a run is waiting to processed</li>
+                        <li>Processing - This is when a reduction is in progress</li>
+                        <li>Complete - The reduction has been complete and the reduced data is available</li>
+                        <li>Skipped - If a run was marked to be skipped and not reduced</li>
+                        <li>Error - Something has gone wrong processing the reduction or in the reduction script</li>
```

Comment:
I like how clear and succinct these are 🙂.
My one minor, pedantic nit pic is: The first two explanation you start with a _'This is when'_ format, but the rest are in different formats. I'd say for consistency we might as well scrap the lead ups altogether. Similarly, to make use of articles (a/the) consistent:
```suggestion
                        <li>Queued - The run is waiting to processed</li>
                        <li>Processing - The reduction is in progress</li>
                        <li>Complete - The reduction has been complete and the reduced data is available</li>
                        <li>Skipped - The run was marked to be skipped and has not been reduced</li>
                        <li>Error - Something has gone wrong processing the reduction or in the reduction script</li>
```

Reply:
Cheers